### ISSUE TITLE
install assembly solver

### ISSUE THREAD

**Issue by bambukouk** [realthunder/FreeCAD_assembly3#1056]
I have now tested it on totally clean/virgin installation (both OS and FreeCAD)
Results below (apart from the fact that FreeCAD got into some loop and couldn't stop it from trying to install...)
In the end after shutting PC down and restarting  all is fine and solver is installed despite dire warnings previously.
I know it has been reported before but I wanted to try it on totally new installation just in case some of what I did previously might have caused problem?

![image](https://user-images.githubusercontent.com/50301871/149938684-e972aac3-6f76-4769-b02a-9659d5a69539.png)
![image](https://user-images.githubusercontent.com/50301871/149938783-9a9bbbe8-bfd3-41ee-8332-f35bf972f9fd.png)

```
OS: elementary OS 6.1 JAlnir (Pantheon/pantheon)
Word size of FreeCAD: 64-bit
Version: 2022.111.30764 (Git) AppImage
Build type: Release
Branch: LinkDaily
Hash: ef3092bcbca5475eff4eb7ff7933eb3e9bbbe4b9
Python version: 3.9.9
Qt version: 5.12.9
Coin version: 4.0.1
OCC version: 7.5.3
Locale: English/United States (en_US)

```

**Comment by luzpaz** [comment]
Any traction on this?

**Comment by bambukouk** [comment]
sorry, can you please elaborate?
not sure I follow your question
not very good at git speak :-)

**Comment by luzpaz** [comment]
'traction' as in 'progress' :wink:  
Just wondering if @realthunder has addressed this yet?

**Comment by bambukouk** [comment]
Thank you 👍 
no, he didn't as far as I know?
It's a bit of "non-issue" because we know that despite warnings the update actually happens, but...
might confuse some (many?)

**Comment by realthunder** [comment]
Yeah, the problem should be solved in 361fde2e67bdcf3fc485669e832e7bfe26c90ed3. I'll do another minor release soon.
